Scripts/SearchIncidentsv2/SearchIncidentsv2.py

```diff
@@ -0,0 +1,60 @@
+from typing import Dict, List, Union
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+
+
+special = ['n', 't', '\\', '"', '\'', '7', 'r']
+
+
+def errors_handel(res: List) -> Union[None, str]:
```

Comment:
please refactor for a more indicative name.
e.g:
errors_handel -> check_empty_response/check_if_found_incident